File: src/main/java/com/gmail/borlandlp/minigamesdtools/lobby/LobbyServerAPI.java
package com.gmail.borlandlp.minigamesdtools.lobby;

import org.bukkit.entity.Player;

public interface LobbyServerAPI {
    ServerLobby getLobbyByPlayer(Player p);
    ServerLobby getLobbyByID(String id);
    void register(ServerLobby l);
    void unregister(ServerLobby l);
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/lobby/LobbyServerAPI.kt
package com.gmail.borlandlp.minigamesdtools.lobby

import org.bukkit.entity.Player

interface LobbyServerAPI {
    fun getLobbyByPlayer(player: Player): ServerLobby?
    fun getLobbyByID(id: String): ServerLobby?
    fun register(serverLobby: ServerLobby)
    fun unregister(serverLobby: ServerLobby)
}